wgpu-core/platform-features/windows-linux-android-emscripten/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
for all proxy crates: should have a doc stub that points at the crate's readme which explains why this exists

Reply:
Done.